chart/templates/cron-jobs/backfill/_container.tpl

```diff
@@ -12,7 +12,6 @@
     {{ include "envCache" . | nindent 2 }}
     {{ include "envQueue" . | nindent 2 }}
     {{ include "envCommon" . | nindent 2 }}
-    {{ include "envMetrics" . | nindent 2 }}
```

Comment:
is envMetrics still used somewhere? Otherwise, we should also remove its template

Reply:
It is still being used in mongodb-migration because it is needed for the last migration (drop collection). I think we can remove it after this PR is merged.